src/main/scala/mesosphere/util/ZookeeperDirectState.scala

```diff
@@ -0,0 +1,186 @@
+package mesosphere.util
+
+import java.lang.Boolean
+import java.util
+import java.util.UUID
+import java.util.concurrent.{ Future => JFuture }
+
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Please don't use this executor. We have one in `mesosphere.util.ThreadPoolContext`